lib/jobs/ansible-job.js

```diff
@@ -107,11 +97,27 @@ function ansibleJobFactory(
                 logger.warning("Result file not found. " + err.message);
                 self._done();
             } else {
-                self._done(err);
+                loadResultFile(self.outFileName,self.context)
+                .then(function() {
+                    self._done(err);
+                });
             }
         });
     };
 
+    function loadResultFile(resultFile,thisContext) {
+        return fs.readFileAsync(resultFile, 'utf8')
+        .then(function(contents) {
+            assert.arrayOfString(thisContext.ansibleResultFile);
+            thisContext.ansibleResultFile.push(contents);
+            fs.unlink(resultFile, function(err) {
```

Comment:
The unlink here I think is meant to be inconsequential to the success/failure of the job.